src/Form/AuthenticationForm.php

```diff
@@ -415,6 +433,37 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     parent::submitForm($form, $form_state);
   }
 
+  /**
+   * Disable form and show error message.
+   *
+   * If there are issues with the private file path, we want to show an error
+   * to the user instead of the form.
+   *
+   * @param array $form
+   *   The form to disable.
+   * @param string $error
+   *   The error to display.
+   */
+  protected function disableForm(array &$form, string $error) {
```

Comment:
The second parameter should be a TranslatableMarkup object because this is what $this->t() returns, you should not cast that to a string here.

Reply:
Good point, updated code and tests to use `TranslatableMarkup`